xpsi/Sample.py

```diff
@@ -1,9 +1,14 @@
-__all__ = ["nested", "ensemble"]
+__all__ = ["nested", "ensemble", "ultranested"]
 
 from xpsi.global_imports import *
 from xpsi import _comm, _rank, _size
 from xpsi.utils import make_verbose
 from xpsi import Likelihood
+from xpsi.Posterior import Posterior
+
+from xpsi.EnsembleSampler import EnsembleSampler
```

Comment:
Ah, I see now the problem is that you changed the location of the emcee import such that now emcee is required to be installed, instead of that being only needed if actually using it.

Reply:
Your new pull request seems to solve this issue.